rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -220,11 +274,16 @@ def get_balances(
                     ).fetchone()
                     if exist_result is None or bool(exist_result[0]) is False:
                         cursor.execute(
-                            'INSERT OR IGNORE INTO nfts('
-                            'identifier, name, last_price, last_price_asset, manual_price, owner_address'  # noqa: E501
-                            ') VALUES(?, ?, ?, ?, ?, ?)',
+                            'INSERT OR REPLACE INTO nfts('
+                            'identifier, name, last_price, last_price_asset, manual_price, owner_address, is_lp, image_url, usd_price, collection_name'  # noqa: E501
+                            ') VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
                             entry,
                         )
+                    else:
+                        cursor.execute(
+                            'UPDATE nfts SET image_url=? WHERE identifier=?',
+                            (entry[7], entry[0]),
```

Comment:
the image is not the only thing that can change, also the price right?

Reply:
You are making too many assumptions here. EVerything can change. NAme, collection name etc. Why even bother with this if/elif?

What's more unlike the `if` part here you do an `UPDATE`. If the nft identifier is not in the DB then this does nothing.
Perhaps you somehow think you know that the id is always in the DB here? I don't see it. The fact that looking at this code for a bit and not seeing it, just like I mentioned above points to this entire thing being very poorly written and I recommend thinking about it, what you want to do here and how you can make it readable.